sdk/core/loader/boot.cc

```diff
@@ -855,6 +855,8 @@ namespace
 
 			// Stack pointer points to the top of the stack.
 			stack.address() += config.stack.size();
+			// Reserve space at the start of the stack for error handling and so on.
+			stack.address() -= 16;
```

Comment:
This `16` needs to agree with the `16` at https://github.com/CHERIoT-Platform/cheriot-rtos/blob/e4ebccc60840de78a919c42e6dd7045c99b68245/sdk/core/switcher/entry.S#L231 .  Can we get a named constant (`#define` is fine)?

Reply:
Done.